### PR TITLE
Add support for compliance_config

### PR DESCRIPTION
**Pull request by daisukixci**
Add support for compliance_config in security-agent.yaml.j2.

This entry is present in https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml#L1581-L1611

### CODE REVIEW

**Review by bkabrda — CHANGES_REQUESTED** [review]
Hi :wave: thanks for the PR. The functionality itself looks fine, but could you please document this new value in the `README.md` section here: https://github.com/DataDog/ansible-datadog#system-probe to make sure it's discoverable? Thanks!

**Comment by daisukixci** [comment]
> Hi wave thanks for the PR. The functionality itself looks fine, but could you please document this new value in the `README.md` section here: https://github.com/DataDog/ansible-datadog#system-probe to make sure it's discoverable? Thanks!

Added, let me know if you need something else

File: README.md [hunk 1]
```diff
@@ -200,6 +200,8 @@ The system probe is configured under the `system_probe_config` variable. Any var
 
 [Universal Service Monitoring][17] (USM) is configured under the `service_monitoring_config` variable. Any variables nested underneath are written to the `system-probe.yaml`, in the `service_monitoring_config` section.
 
+[Compliance][] is configured under the `compliance_config` variable. Any variables nested underneath are written to the `security-agent.yaml`, in the `compliance_config` section.
```
Comment (bkabrda) [inline README.md]:
```suggestion
[Compliance][18] is configured under the `compliance_config` variable. Any variables nested underneath are written to the `security-agent.yaml`, in the `compliance_config` section.
```

**Review by bkabrda — CHANGES_REQUESTED** [review]
:wave: just one minor suggestion to address, then I can approve and merge. Thanks!

File: README.md [hunk 2]
```diff
@@ -114,11 +114,11 @@ To configure a custom check use the configuration below. This creates the corres
 
 ##### Custom Python Checks
 
-To pass a Python check to the playbook, use the configuration below. 
+To pass a Python check to the playbook, use the configuration below.
 
 This configuration requires the Datadog [play and role][12] to be a part of the larger playbook where the value passed in is the relative file path to the actual task for [Linux][13] or [Windows][14].
 
-This is only available for Agent v6+.  
+This is only available for Agent v6+.
```
Comment (alai97) [inline README.md]:
```suggestion
This is only available for Agent v6 or later.
```

**Review by alai97 — APPROVED** [review]
Looks good, one suggestion!

**Review by bkabrda — APPROVED** [review]

**Pull request merged**